modules/ROOT/partials/configuration/template_selected_content_classes.adoc

```diff
@@ -14,6 +14,8 @@ Default Value: `+'selcontent'+`
 tinymce.init({
   selector: 'textarea',  // change this value according to your HTML
   plugins: 'template',
+  menubar: 'insert',
```

Comment:
```suggestion
```
I suspect this is in other template examples, if so it should be removed as including this is more likely to break things than improve it.